### ISSUE TITLE
Annotations in class broken?

### ISSUE THREAD

**Issue by henryiii** [utgwkk/pytest-github-actions-annotate-failures#8]
I've tried adding this plugin:

https://github.com/scikit-hep/boost-histogram/pull/447/checks?check_run_id=1000258530

And I get:

```
##[error]self = <test_axis.TestRegular object at 0x7f907321b520>
```

Which doesn't help find the line of the test (though it's slightly better than not having the red error line). Would this be fixable?

Edit: Also parametrized tests. Which is pretty much most of my tests. :'(

Edit: Actually, it doesn't seem to be outputting anything. Is there something that needs configuring?

Edit yet again: Locally, this works. So maybe I just don't know where to look for the failure annotations?

**Comment by henryiii** [comment]
Ahah, maybe I found the issue? It seems that it reports the beginning of the function the failure was in, instead of the line that failed, and the annotation don't show up then, since only part of the function changed, not the whole function, and GitHub requires that something changes to show the annotation. Or I could be completely wrong.

**Comment by utgwkk** [comment]
Looks like the plugin made annotations for `../tests/test_axis.py`. You ran tests in `build` directory and the plugin didn't resolve relative paths!
This is a bug, so I'll fix it.

**Comment by utgwkk** [comment]
It will be fixed with #11 and #12

**Comment by utgwkk** [comment]
I've just released [version 0.0.8](https://pypi.org/project/pytest-github-actions-annotate-failures/0.0.8/) containg fix of it, thanks!
